docs/code-of-conduct.md

```diff
@@ -69,8 +69,8 @@ dispute. If you are unable to resolve the matter for any reason, or if the
 behavior is threatening or harassing, report it. We are dedicated to providing
 an environment where participants feel welcome and safe.
 
-Reports should be directed to *[PROJECT STEWARD NAME(s) AND EMAIL(s)]*, the
-Project Steward(s) for *[PROJECT NAME]*. It is the Project Steward’s duty to
+Reports should be directed to _[PROJECT STEWARD NAME(s) AND EMAIL(s)]_, the
+Project Steward(s) for _[PROJECT NAME]_. It is the Project Steward’s duty to
```

Comment:
I think I missed this in the initial templating.  This should probably have `Jeff Kaufman` / `jefftk@google.com` / `fledge-shim`.

Reply:
You could fix now, or I can send a follow up